<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(206, 206, 206);</string>
    </property>
    <property name="text">
     <string>Reset window</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>801</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="MainLayout">
     <item row="2" column="2">
      <widget class="QTextEdit" name="textOut"/>
     </item>
     <item row="0" column="2">
      <widget class="QTextEdit" name="textInput"/>
     </item>
     <item row="4" column="2">
      <layout class="QGridLayout" name="LayoutForCount">
       <item row="2" column="1">
        <widget class="QLCDNumber" name="valueSymbolsWithoutSpace">
         <property name="styleSheet">
          <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 244, 221);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLCDNumber" name="valueWords">
         <property name="styleSheet">
          <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 244, 221);</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLCDNumber" name="valueSymbols">
         <property name="acceptDrops">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 8pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 244, 221);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="segmentStyle">
          <enum>QLCDNumber::Filled</enum>
         </property>
         <property name="intValue" stdset="0">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="countWords">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 244, 221);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Слов в тексте</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="countSymbols">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 244, 221);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Символов в тексте</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="countSymbolsWithoutSpace">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 244, 221);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Символов без  пробелов</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="selectMode">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(206, 206, 206);</string>
    </property>
    <item>
     <property name="text">
      <string>Подсчет текста</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Изменение раскладки</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(206, 206, 206);</string>
    </property>
    <property name="text">
     <string>Введите текст</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuToxic">
    <property name="title">
     <string>Toxic</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menuToxic"/>
  </widget>
  <action name="actionMAin">
   <property name="text">
    <string>MAin</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Тут пока ничего нет</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ResetButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
